packages/core/src/amazonq/activation.ts

```diff
@@ -53,14 +46,3 @@ function registerApps(appInitContext: AmazonQAppInitContext) {
     featureDevChatAppInit(appInitContext)
     gumbyChatAppInit(appInitContext)
 }
-
-export const amazonQWelcomeCommand = Commands.declare(
-    { id: 'aws.amazonq.welcome', compositeKey: { 1: 'source' } },
-    (context: ExtensionContext, publisher: MessagePublisher<any>) => (_: VsCodeCommandArg, source: string) => {
-        telemetry.ui_click.run(() => {
-            void focusAmazonQPanel()
-            welcome(context, publisher)
```

Comment:
The welcome() function is no longer needed. Should it be deleted?

Also, the the welcome page involved a bunch of resources (svg files, etc), generally everything in `src/amazonq/onboardingPage/` . So, is any of that still needed, and why?